aea/cli/registry/fetch.py

```diff
@@ -49,14 +54,17 @@ def fetch_agent(ctx: Context, public_id: PublicId, alias: Optional[str] = None)
 
     filepath = download_file(file_url, ctx.cwd)
```

Comment:
please, add the description of `target_dir` in the docstring (although it is pretty self-describing)